content-src/components/TopSites/TopSite.jsx

```diff
@@ -493,6 +506,19 @@ export class TopSiteList extends React.PureComponent {
         break;
       case "drop":
         if (index !== this.state.draggedIndex) {
+          if (this.state.draggedSite.type === "SPOC") {
```

Comment:
@Mardak @punamdahiya - This will be deleted! I initially had this because I was storing image URLs in tippytopicon and we didn't handle that in insert.